pkg/kubectl/cmd/set/set_image.go

```diff
@@ -43,13 +43,12 @@ type SetImageOptions struct {
 	PrintFlags  *printers.PrintFlags
 	RecordFlags *genericclioptions.RecordFlags
 
-	Infos        []*resource.Info
-	Selector     string
-	DryRun       bool
-	All          bool
-	Output       string
-	Local        bool
-	ResolveImage func(in string) (string, error)
```

Comment:
This is good for unit testing.  Don't lose it.

Reply:
Done - reverted changes to set_image.go